client/src/components/v3/StatisticRow.tsx

```diff
@@ -26,13 +26,17 @@ export default function StatisticRow(): JSX.Element {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [currVel]);
 
+  // Previous Trap Speed (Achieved)
+  // eslint-disable-next-line no-unused-vars
```

Comment:
we don't require this `ESLint-disable-next-line` stuff cos we now do use the `setPrevTrapSpeed` func. Previously we had it defined but never called it and hence we got the `no-unused-vars` error from ESLint, and had to disable it.